packages/tools/pie-icons-webc/test/helpers/page-object/selectors.ts

```diff
@@ -0,0 +1,15 @@
+const icons = {
+    selectors: {
+        alcoholFilled: {
+            description: 'The selector for the alcohol filled icon',
```

Comment:
Not a blocker, but I would like to discuss these a bit more at some point. I question the value of some of these descriptions. I don't think these selectors would be exported for external use because they are only related to the automated test stories, and sometimes the names are self-explanatory and we do not use them in the tests, so I wonder if they should be optional.

I am all for consistency, but in my mind an optional description properly _is_ consistent, it's just a more flexible schema

Reply:
Yep happy to discuss this. Agree that selectors don't have value being exported _yet_. My hope though, is that in the future, other teams leveraging our components can use these selectors as a source of truth for use in their own test frameworks, preventing a situation changes to component structure can break our consumers tests; something that unfortunately I've seen too often in other repos i've worked in.

As mentioned in conversation, I think there's more that can be done to improve this schema. Annoyingly, I found a repo years ago that offered a really nice opinionated schema for POM, and unfortunately, I'm unable to find it. Will of course bring it up with the team if I ever come across it again!